### ISSUE TITLE
twitter.com: breakage

### ISSUE THREAD

**Issue by binarymaster** [uBlockOrigin/uAssets#24018]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Search [here](https://github.com/uBlockOrigin/uAssets/issues) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL(s) where the issue occurs.

```yaml
https://twitter.com/notifications
https://twitter.com/home
https://twitter.com/messages
https://twitter.com/explore
```

### Description

This problem is happening to me for two weeks or so. I'm using Microsoft Edge 125.0.2535.92.

I've verified that the problem is caused by "uBlock filters – Ads" filters, when I disable that list, Twitter starts to load pages normally. Disabling other extensions does not affect this in any way.

Also it seems the same issue was reported for Brave browser in their community: https://community.brave.com/t/shield-and-ublock-breaking-twitter-on-desktop-brave/548199

### Other extensions used

- Browsec VPN
- Tampermonkey
- Requestly

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/578406/50602b5e-e758-47cd-9b6a-2d1331db1fc3)
![image](https://github.com/uBlockOrigin/uAssets/assets/578406/6d74bce1-06fd-4ec3-998a-2c1585777ba9)
![image](https://github.com/uBlockOrigin/uAssets/assets/578406/7bc97eb3-4683-465f-b5bc-f7a2f8a0d658)

</details>

### Configuration

<details>
<summary>Details</summary>

```yaml
uBlock Origin: 1.58.0
Chromium: 125
filterset (summary):
 network: 157064
 cosmetic: 67067
 scriptlet: 22462
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 38341-271, 16m
  ublock-badware: 9216-0, 21m
  ublock-privacy: 928-0, 21m
  ublock-unbreak: 2336-0, 21m
  ublock-quick-fixes: 78-0, 21m
  easylist: 86618-587, 22m
  easyprivacy: 51022-23, 22m
  urlhaus-1: 16720-0, now
  plowe-0: 3555-615, now
  RUS-0: 39557-121, now
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 5321 ms
 maxAssetCacheWait: 1326 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 1
 network:
  ads-twitter.com: 1
 extended:
  ##+js(json-prune-xhr-response, data.home.home_timeline_urt.instr…
  ##+js(json-prune-xhr-response, data.search_by_raw_query.search_t…
  ##+js(json-prune-xhr-response, data.threaded_conversation_with_i…
  ##+js(json-prune-xhr-response, data.user.result.timeline_v2.time…
```
</details>

**Comment by binarymaster** [comment]
> unable to reproduce

A logged in account is obviously needed.

If there's anything I can try to test, let me know.

**Comment by stephenhawk8054** [comment]
Test again in a new browser profile with only uBO and its default settings

**Comment by binarymaster** [comment]
Indeed, creating a new profile solves the problem, here's debug info from it:

```yml
uBlock Origin: 1.58.0
Chromium: 125
filterset (summary):
 network: 154582
 cosmetic: 67083
 scriptlet: 22478
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 38375-273, 4m
  ublock-badware: 9226-0, 4m
  ublock-privacy: 929-0, 4m
  ublock-unbreak: 2337-0, 4m
  ublock-quick-fixes: 82-0, 4m
  easylist: 86628-587, 4m
  easyprivacy: 51023-23, 5m
  urlhaus-1: 14210-0, 5m
  plowe-0: 3555-615, 5m
  RUS-0: 39558-121, 5m
filterset (user): [empty]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 34854 ms
 maxAssetCacheWait: 24 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 20
 network:
  twitter.com: 19
  ads-twitter.com: 1
 extended:
  ##+js(json-prune-xhr-response, data.home.home_timeline_urt.instr…
  ##+js(json-prune-xhr-response, data.search_by_raw_query.search_t…
  ##+js(json-prune-xhr-response, data.threaded_conversation_with_i…
  ##+js(json-prune-xhr-response, data.user.result.timeline_v2.time…
```

Is there anything I can do to fix or analyze the problem in my current profile?

**Comment by stephenhawk8054** [comment]
Can you go to `chrome://extensions` page (type in URL bar) and screenshot it here?

**Comment by binarymaster** [comment]
Hey! It looks like I found that the guilty extension is Requestly 24.5.8 -- I'm pretty sure I have already tried disabling it, but now the behavior is clear - enabling it reproduces the problem, and disabling it makes the problem go away.

Note that I have all HTTP filters disabled in that extension.

Also, installing Requestly in the new browser profile 2 does not introduce the problem... probably because the extension version is different there -- it's 24.5.27 (MV3).

So I conclude that the problem is conflicting extension, specifically Requestly version 24.5.8 in my case. Let me know if you want to analyze the problem deeper.

<details>
<summary>Screenshot(s)</summary>




![image](https://github.com/uBlockOrigin/uAssets/assets/578406/d41ef031-65b8-4f33-929e-2ca6c880f85b)
![image](https://github.com/uBlockOrigin/uAssets/assets/578406/ab61cf1e-8ae4-4221-937f-a700604bbb94)



</details>
